dolphinscheduler-server/src/main/java/org/apache/dolphinscheduler/server/utils/ParamUtils.java

```diff
@@ -87,49 +91,67 @@
                  *  and there are no variables in them.
                  */
                 String val = property.getValue();
-                val  = ParameterUtils.convertParameterPlaceholders(val, timeParams);
+                val = ParameterUtils.convertParameterPlaceholders(val, timeParams);
                 property.setValue(val);
             }
         }
 
+        // set the schedule time
```

Comment:
Thanks for your contribution. I don't think the spark parameter can be modified here, which will affect other components, please check